app/src/lib/stores/pull-request-store.ts

```diff
@@ -139,9 +142,16 @@ export class PullRequestStore {
     api: API,
     repository: GitHubRepository
   ) {
-    const { name, owner } = getNameWithOwner(repository)
-    const open = await api.fetchAllOpenPullRequests(owner, name)
-    await this.storePullRequestsAndEmitUpdate(open, repository)
+    const bothPrLists = await Promise.all([
+      fetchOpenPullRequests(api, repository),
+      repository.fork && repository.parent
+        ? fetchOpenPullRequests(api, repository)
```

Comment:
Maybe I'm missing something, but shouldn't this fetch the open PRs on `repository.parent` not `repository`?